relay/polkadot/src/lib.rs

```diff
@@ -1661,8 +1691,8 @@ pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 pub type SignedBlock = generic::SignedBlock<Block>;
 /// `BlockId` type as expected by this runtime.
 pub type BlockId = generic::BlockId<Block>;
-/// The `SignedExtension` to the basic transaction logic.
-pub type SignedExtra = (
+/// The `TransactionExtension` to the basic transaction logic.
+pub type TxExtension = (
```

Comment:
We should add `frame_system::WeightReclaim<Runtime>,` at the end of the transaction extension pipeline, so the unspent weight in transaction extensions is reclaimed correctly.

Reply:
See https://github.com/polkadot-fellows/runtimes/pull/606#discussion_r2029078959